src/styles/_vars.scss

```diff
@@ -0,0 +1,6 @@
+$clock-size: 400px;
+$arrow-h: $clock-size * 0.55;
+$arrow-w: $clock-size * 0.02;
+$arrow-short-h: $clock-size * 0.65;
+$arrow-short-w: $clock-size * 0.02;
+$top-space: $clock-size / 2;
```

Comment:
![image](https://user-images.githubusercontent.com/43670243/169381745-471bd46b-cd3a-4e73-ac74-a2fa1c09f79c.png)

at some point, the clock looks a bit odd

Reply:
![image](https://user-images.githubusercontent.com/43670243/169385326-8ea65a1f-1886-4a5d-9f57-48f5de97415d.png)

transform-origin is a tricky thing